src/unix/linux_like/android/mod.rs

```diff
@@ -528,16 +526,16 @@ s_no_extra_traits! {
     }
 
     pub struct dirent {
-        pub d_ino: u64,
-        pub d_off: i64,
+        pub d_ino: crate::ino_t,
+        pub d_off: crate::off_t,
```

Comment:
This either needs to remain unmodified or become `off64_t` - it is [unconditionally 64-bits on android](https://cs.android.com/android/platform/superproject/main/+/main:bionic/libc/include/dirent.h;l=69?q=d_ino&ss=android%2Fplatform%2Fsuperproject%2Fmain:bionic%2Flibc%2Finclude%2F&start=1)